data-raw/ccao_ids.R

```diff
@@ -35,6 +38,7 @@ left <- c(
   "epic",
   "exciting",
   "fancy-free",
+  "fecund",
```

Comment:
**[Nitpick, non-blocking]** I'm on the fence about this one 😅 From the [top-level OED summary](https://www.oed.com/search/dictionary/?scope=Entries&q=fecund):

> fecund, adj.
> Of animals, the earth, etc.: Capable of producing offspring or vegetable growth abundantly; prolific, fertile.

According to [Merriam-Webster](https://www.merriam-webster.com/dictionary/fecund), "prolific" is the most apt synonym, but I feel hesitant about the connection to "fertile" in the OED definition!